management/src/test/resources/clusteredConfiguration.txt

```diff
@@ -0,0 +1,24 @@
+caches:
+    cache-1:
+        keyType: java.lang.String
+        valueType: java.lang.String
+        serviceConfigurations:
+        evictionAdvisor:
+        expiry: NoExpiry
+        resourcePools:
+            pools:
+                heap:
+                    size: 10 entries
+                    tierHeight: 10000
+                offheap:
+                    size: 1 MB
+                    tierHeight: 1000
+                clustered-dedicated:
+                    size: 1 MB (persistent)
+                    tierHeight: 10
+services:
+    - org.ehcache.clustered.client.config.ClusteringServiceConfiguration:
+        clusterUri: passthrough://server-1:9510/my-server-entity-1
+        readOperationTimeout: null
```

Comment:
Ignore the `null` here, that is a bug - see #1344 
However, I stand by what I said earlier ... as it will end up being MAX_LONG <unit>, this should be reflected instead of having the explicit long printed there.